met-web/src/components/engagement/listing/AdvancedSearch/SearchComponent.tsx

```diff
@@ -120,6 +118,9 @@ const AdvancedSearch: React.FC<filterParams> = ({ setFilterParams }) => {
                                         onChange={handleStatusFilterChange}
                                         value={EngagementDisplayStatus[EngagementDisplayStatus.Draft]}
                                         checked={statusFilter[EngagementDisplayStatus[EngagementDisplayStatus.Draft]]}
+                                        style={{
```

Comment:
use the sx prop instead of the style prop please

Reply:
updated, thanks.